README.md

```diff
@@ -189,21 +189,27 @@ public class YourInterceptor
 
 ## Running locally
 
-The easiest way to run this server entirely depends on your environment requirements. At least, the following 4 ways are supported:
+The easiest way to run this server entirely depends on your environment requirements. The following ways are supported:
 
-### Using jetty
+### Using Spring Boot
 ```bash
-mvn -Pjetty jetty:run
+mvn spring-boot:run
 ```
 
+The Server will then be accessible at http://localhost:8080/fhir and the CapabilityStatement will be found at http://localhost:8080/fhir/metadata.
+
+### Using jetty
+```bash
+mvn -Pjetty spring-boot:run
+```
 
 If you need to run this server on a different port (using Maven), you can change the port in the run command as follows:
 
 ```bash
-mvn -Pjetty -Djetty.port=8888 jetty:run
+mvn -Pjetty -Dserver.port=8888 spring-boot:run
```

Comment:
This doesn't appear to work as advertised. If I run in using these params, I still get the server on port 8080.

Reply:
@dotasek   Yes, that is why I haven't finished work on this PR yet. That parameter is supposed to work, but it doesn't have any effect. I will keep investigating this to try and figure it out.

Reply:
Not sure if helpful, but in my deployments of HAPI I set the port in the application.yml like this and it works: 

```
server: 
  servlet: 
    context-path : /
  port: 8443
```

If that does work, I might suggest just adding that section the default property file (using 8080 instead of 8443 obviously).

Reply:
I misread that the problem is for Jetty, which I'm not using. 
But I notice it might be server.servlet.port and not just server.port.

Reply:
@XcrigX   I have updated the instructions and added an example as per your recommendation...thanks for that!